TangutIDS.txt

```diff
@@ -6143,3 +6151,25 @@
 18D06	𘴆	⿰𘫿𘦳
 18D07	𘴇	⿰𘧤𘤊
 18D08	𘴈	⿰𘪡𘤊
+18D09	𘴉	⿰𘩍𘧓
+18D0A	𘴊	⿰⿰𘫽𘠨⿲𘢌𘠁𘤘
+18D0B	𘴋	⿰𘫽𘫾
```

Comment:
The [WG2 N5217R2](https://unicode.org/wg2/docs/n5217r2-Tangut-components-ideographs.pdf) gives ⿰𘤊𘤕, however, this character is likely a variant of 𗿠, whose IDS is ⿰𘫽⿱𘣍𘫾. For consistency we replace 𘤊 (Component 267) by 𘫽 (Component 766).